lib/test_runner_test.go

```diff
@@ -54,34 +50,35 @@ func newTestRunner(t *testing.T) *testRunner {
 		t.Fatal(err)
 	}
 
-	bus := event.NewBus(ctx)
-
 	// A temporary directory for doing filesystem work.
 	tmpDir, err := ioutil.TempDir("", "lib_test_runner")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	mr, err := testrepo.NewEmptyTestRepo(bus)
-	if err != nil {
-		t.Fatalf("error allocating test repo: %s", err.Error())
-	}
-	localResolver := dsref.SequentialResolver(mr.Dscache(), mr)
-	node, err := p2p.NewQriNode(mr, testcfg.DefaultP2PForTesting(), bus, localResolver)
-	if err != nil {
-		t.Fatal(err.Error())
+	qriPath := filepath.Join(tmpDir, "qri")
+	if err := os.Mkdir(qriPath, os.ModePerm); err != nil {
+		t.Fatal(err)
```

Comment:
Ensures we have write permissions, `ioutil.TempDir` doesn't guarantee anything.

Reply:
_faaaaaancy_